### PR TITLE
Update charts.d.plugin yaml

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

add minimal info to charts.d.plugin sensors module

### CODE REVIEW

File: collectors/charts.d.plugin/sensors/metadata.yaml [hunk 1]
```diff
@@ -2,20 +2,26 @@ meta:
...
+    - sensors
+    - sysfs
+    - hwmon
+    - rpi
+    - raspberry pi
   most_popular: false
 overview:
   data_collection:
-    metrics_description: ''
+    metrics_description: "Examine Linux Sensors metrics with Netdata for insights into hardware health and performance. Enhance your system's reliability with real-time hardware health insights."
```
Comment (thiagoftsm) [inline collectors/charts.d.plugin/sensors/metadata.yaml:24]: @Ancairon , please, take a look in next warning.

Comment (Ancairon) [reply collectors/charts.d.plugin/sensors/metadata.yaml:24]: After a talk with Austin, we figured it is safe to ignore for now

**Review by thiagoftsm — APPROVED** [review]
Considering answer received, LGTM!

**Review by Dim-P — APPROVED** [review]
I'm a bit in the dark about how keywords are going to be used, but how about temperature, fans etc.?

Otherwise looks good so I'm approving it.

**Comment by Ancairon** [comment]
I think it would be best to merge now, and take note of your suggestions for future iterations, for sure, once the yaml files are complete the keyword list should have terms as you suggest.

**Pull request merged**